prov/efa/src/rxr/rxr_init.c

```diff
@@ -83,12 +83,31 @@ struct rxr_env rxr_env = {
 	.rnr_retry = EFA_RNR_INFINITE_RETRY,
 };
 
+
+static int efa_convert_mem_monitor_str(const char *value)
+{
+        int i;
+
+        if (!value) {
+                return EFA_UFFD_MONITOR;
+	}
+
+        for (i = 0; efa_mem_monitors[i]; i++) {
+                if (!strcasecmp(value, efa_mem_monitors[i]))
+                        return i;
+        }
+        return EFA_UFFD_MONITOR;
```

Comment:
If user set something we do not understand, we should fail and print warning, instead of set the default value, because it is possible user has a typo in the environment variable.